packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -5040,7 +5040,7 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
 class _ElementDiagnosticableTreeNode extends DiagnosticableTreeNode {
   _ElementDiagnosticableTreeNode({
     super.name,
-    required Element super.value,
```

Comment:
Prior to your change, only `Element`s were acceptable as `value`, now any `DiagnosticableTree` can be passed in. Since this is explicitly called `_ElementDiagnosticableTreeNode` tightening the type to `Element` seems to be the right thing to do. See also line 5053 where value is assumed to be an Element. We should be explicit about this in the constructor.